pkgs/development/tools/ko/default.nix

```
@@ -7,16 +7,20 @@
 
 buildGoModule rec {
   pname = "ko";
-  version = "0.13.0";
+  version = "0.14.1";
 
   src = fetchFromGitHub {
     owner = "ko-build";
     repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-KVJqqvp46BAUscG5Xj/g4ThUXKFsuJdzEB++uBskFiw=";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-jBysfeoZ9W94c07xFODBASrWGJbZRHsUODfEul9f4Ug=";
   };
 
-  vendorSha256 = null;
+  patches = [
+    ./skip-failing-tests.patch
+  ];
```

this worked for me

```suggestion
  checkFlags = [
    # requires docker daemon
    "-skip=TestNewPublisherCanPublish"
  ];
```

ideally it should be below ldflags, but github doesn't let me review there